pkg/postage/postagecontract/contract.go

```diff
@@ -199,13 +199,7 @@ func (c *postageContract) sendTransaction(ctx context.Context, callData []byte,
 	}
 
 	if receipt.Status == 0 {
-		err := transaction.ErrTransactionReverted
-		if res, cErr := c.transactionService.Call(ctx, request); cErr == nil {
-			if reason, uErr := abi.UnpackRevert(res); uErr == nil {
-				err = fmt.Errorf("%w: reason: %s", err, reason)
-			}
-		}
-		return nil, err
+		return nil, c.transactionService.UnwrapRevertReason(ctx, request, transaction.ErrTransactionReverted)
```

Comment:
could be deferred at the starting of the function such as in redistribution.go

Reply:
Done.